src/main/java/com/meilisearch/sdk/model/Result.java

```diff
@@ -1,21 +1,11 @@
 package com.meilisearch.sdk.model;
 
-import com.google.gson.Gson;
 import lombok.Getter;
 
 /** Data structure of MeiliSearch response for a Result */
+@Getter
```

Comment:
Why do we have a `@Getter` here?

Reply:
To be able to get the `Result` who should be `Results` by the way ...
```java
    public void testClientGetKeys() throws Exception {
        Result<Key> result = client.getKeys();
        Key[] keys = result.getResults();
```